suzieq/engines/pandas/network.py

```diff
@@ -157,39 +158,31 @@ def _find_addr_arp(self, addr: str, **kwargs) -> pd.DataFrame:
             else:
                 oif = row.oif
 
-            active = getattr(row, 'active', True)
-            if not active:
-                result.append({
-                    'active': False,
-                    'namespace': row.namespace,
-                    'hostname': row.hostname,
-                    'vrf': '-',
-                    'ipAddress': row.ipAddress,
-                    'vlan': '-',
-                    'macaddr': row.macaddr,
-                    'ifname': row.oif,
-                    'type': 'bridged',
-                    'l2miss': False,
-                    'timestamp': row.timestamp
-                })
-                continue
+            active = getattr(row, 'active', False)
 
-            ifdf = self._get_table_sqobj('interfaces') \
+            # We ignore the start time because the interface may not have
+            # had a change in a very long time. Using the start-time if
+            # one is provided can make us not return the relevant data
+            # in situations where the interface has been stable long before
+            # the start time.
+            ifdf = self._get_table_sqobj('interfaces', start_time='') \
                 .get(namespace=[row.namespace], hostname=[row.hostname],
                      ifname=[oif])
 
             if not ifdf.empty:
+                row_vrf = ifdf.master.unique().tolist()[0] or 'default'
                 if 'vlan' not in ifdf.type.unique():
                     # Routed interface
                     if hasattr(row, 'active'):
                         tmpres['active'] = True  # False case handled above
 
                     tmpres.update({
+                        'active': active,
                         'namespace': row.namespace,
                         'hostname': row.hostname,
-                        'vrf': ifdf.master.unique().tolist()[0] or 'default',
+                        'vrf': row_vrf,
                         'ipAddress': row.ipAddress,
-                        'vlan': ifdf.vlan.astype(str).unique().tolist()[0],
+                        'vlan': ifdf.vlan.unique().tolist()[0],
```

Comment:
Store the list of unique vlans in variable. This value is used in multiple places